packages/rest/src/resource.ts

```diff
@@ -7,6 +7,7 @@ import {
   Model,
```

Comment:
Can we remove the exception in the library-linter/src/linter.ts for that decorator too

Reply:
Will do!